data/sql/updates/pending_db_world/rev_1636980654557070200.sql

```diff
@@ -0,0 +1,50 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1636980654557070200');
+
+DROP TABLE IF EXISTS `player_loot_template`;
+CREATE TABLE IF NOT EXISTS `player_loot_template` (
+  `Entry` MEDIUMINT UNSIGNED NOT NULL DEFAULT 0,
```

Comment:
can be tinyint

Reply:
I'm just afraid that if it's not exactly the same as the other tables we'll have issues during load (the table needs to be an exact copy of the others otherwise creating a entire loading method just for this table would be required :P )